server/templates/base.html

```diff
@@ -49,7 +49,7 @@
       <nav class="navbar navbar-dark navbar-expand-lg col" id="main-nav">
         <div class="container-fluid">
           <div class="navbar-brand">
-            <a href="{{ url_for('static.homepage') }}">Data Commons</a>
+            <a href="{{ url_for('static.homepage') }}">{% if DEPLOYMENT == "private" %} International Energy Agency {% else %} Data Commons {% endif %}</a>
```

Comment:
ok for now, but we should pass the name in as a variable, easiest through a global (we could set the name Data Commons or the private instance name through a single global, so there doesn't need to be a check for DEPLOYMENT)

Reply:
Good point! Moved this and "private" to configmodule.py as variables.